pkg/providers/cloudstack/config/template-md.yaml

```diff
@@ -73,7 +85,7 @@ kind: MachineDeployment
 metadata:
   labels:
     cluster.x-k8s.io/cluster-name: {{.clusterName}}
-  name: {{.clusterName}}-md-0
```

Comment:
md name should be "workerNodeGroupName" instead of `workloadkubeadmconfigTemplateName`